libfwupdplugin/fu-sbatlevel-section.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2023 Canonical Ltd
+// SPDX-License-Identifier: LGPL-2.1+
+
+#[derive(Parse, Validate)]
+struct SbatLevelSectionHeader {
+    version: u32: const=0x0,
+    previous: u32: default=0x0,
+    latest: u32: default=0x0,
```

Comment:
helpfully i've changed the format in https://github.com/fwupd/fwupd/commit/1d16c78c912f504fd0d0e371f4e63fe67132e0b5 -- const is ` == ` default is ` =` - and you don't need a default of 0x0 as that's the _default default_ anyway :)